src/ansys/geometry/core/designer/body.py

```diff
@@ -757,7 +757,7 @@ def fill_style(self, value: FillStyle):  # noqa: D102
     def color(self) -> str:  # noqa: D102
         """Get the current color of the body."""
         if self._color is None:
-            if self._grpc_client.backend_version < (25, 1, 0):
+            if self._grpc_client.backend_version < (25, 1, 0): # pragma: no cover
```

Comment:
why pragma: no cover here? @RobPasMue

Reply:
We won't be able to check coverage here since this is for older versions and we are not actively testing against them